src/utils/wrappers.ts

```diff
@@ -1,22 +1,74 @@
-import { Request, Response, NextFunction } from 'express';
-import { ApplicationError } from './errors/application.error';
+import * as apm from 'elastic-apm-node';
+import { NextFunction, Request, Response } from 'express';
+import { apmConfig } from '../config';
+import { ApplicationError, ClientError } from './errors/application.error';
+
+function extractTraceparent(req: Request): string | undefined {
+  const traceparent = req.header(apmConfig.traceParentHeader); // add as config
+  return traceparent;
+}
```

Comment:
```suggestion
function extractTraceparent(req: Request): string | undefined => {
   req.header(apmConfig.traceParentHeader); // add as config
}
```
This syntax is better for eslint.

Reply:
changed the style.